src/routes/api/bookings/createBooking.ts

```diff
@@ -3,12 +3,14 @@ import timeBlockModel from '../../../models/timeBlock.model';
 import sectionModel from '../../../models/section.model';
 import Room from '../../../models/room.model';
 import { DateTime } from 'luxon';
+import booking from '../../../models/booking.model';
+import mongoose from 'mongoose';
 
-const bookSectionRoute = async (req: Request, res: Response): Promise<void> => {
-    const { id, emails, startsAt, endsAt } = req.body;
+const createBookingRoute = async (req: Request, res: Response): Promise<void> => {
+    const { id, emails, booker, startsAt, endsAt } = req.body;
```

Comment:
I would use the user processed by the authMiddleware for `booker` rather than trusting users to send correct data.